test/language/eval-code/indirect/var-env-var-non-strict.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-performeval
+es5id: 10.4.2-2-c-1
```

Comment:
Yes. At the moment, we're still considering using that tag as a indicator that the test applies to ES5: https://github.com/tc39/test262/issues/569. As I mentioned there, I'd like to simplify that requirement, but in the mean time, I decided to just include the es5id.